transformation/src/main/scala/org/broadinstitute/monster/hca/HcaPipelineBuilder.scala

```diff
@@ -150,6 +160,28 @@ object HcaPipelineBuilder extends PipelineBuilder[Args] {
     )
   }
 
+  /**
+    *
+    * Extract the necessary info from a file of table relationships and put it into a form that
+    * makes it easy to pass in to the table format
+    *
+    * @param fileName the raw filename of the metadata file
+    * @param metadata the content of the metadata file in Msg format
+    * @return a Msg object in the desired output format
+    */
+  def transformLinksFileMetadata(filename: String, metadata: Msg): Msg = {
+    val linksFileName = metadata.read[String]("file_core", filename) //TODO do I need to read from metadata here?
+    val (linksId, linksVersion, projectId) = getLinksIdVersionAndProjectId(linksFileName)
+    // put values in the form we want
+    Obj(
+      mutable.LinkedHashMap[Msg, Msg](
+        Str("links_id") -> Str(linksId),
+        Str("links_version") -> Str(linksVersion),
+        Str("project_id") -> Str(projectId)
+      )
```

Comment:
This still needs a `content` field, which will contain the JSONified value of `metadata`. See how `transformFileMetadata` does it above